urbanopt-geojson-gem.gemspec

```diff
@@ -11,7 +11,8 @@ Gem::Specification.new do |spec|
 
   spec.summary       = "Library and measures to translate URBANopt GeoJSON format to OpenStudio"
   spec.description   = "Library and measures to translate URBANopt GeoJSON format to OpenStudio"
-  spec.homepage      = "https://github.com/urbanopt"
+  spec.files            = ["lib/urbanopt/geojson.rb"]
```

Comment:
Probably better to do a glob so we just grab all files:

```
  spec.files         = Dir.chdir(File.expand_path(__dir__)) do
    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
  end
```

https://github.com/ladybug-tools-in2/energy-model-measure/blob/develop/ladybug-energy-model-measure.gemspec#L18